File: app/src/main/java/nl/rsdt/japp/jotial/maps/wrapper/google/GoogleCameraPosition.java
package nl.rsdt.japp.jotial.maps.wrapper.google;

import nl.rsdt.japp.jotial.maps.wrapper.ICameraPosition;

/**
 * Created by mattijn on 17/08/17.
 */

public class GoogleCameraPosition implements ICameraPosition {

    private com.google.android.gms.maps.model.CameraPosition googlePos;

    public GoogleCameraPosition(com.google.android.gms.maps.model.CameraPosition cameraPosition) {
        googlePos = cameraPosition;
    }

    @Override
    public float getZoom() {
        return googlePos.zoom;
    }

    @Override
    public float getTilt() {
        return googlePos.tilt;
    }
}

File: app/src/main/java/nl/rsdt/japp/jotial/maps/wrapper/google/GoogleCameraPosition.kt
package nl.rsdt.japp.jotial.maps.wrapper.google

import nl.rsdt.japp.jotial.maps.wrapper.ICameraPosition

/**
 * Created by mattijn on 17/08/17.
 */

class GoogleCameraPosition(private val googlePos: com.google.android.gms.maps.model.CameraPosition) : ICameraPosition {

    override val zoom: Float
        get() = googlePos.zoom

    override val tilt: Float
        get() = googlePos.tilt
}
